src/azure-cli/HISTORY.rst

```diff
@@ -11,6 +11,10 @@ Release History
 
 * Fix #10151 - `az ad app update --homepage` not updating homepage.
 
+**Network**
+
+* az network lb create/frontend-ip create: Fixes #10018. Support `--private-ip-address-version` argument to create IPv6 based private-ip-address 
```

Comment:
If you can create --private-ip-address-version, should we add this property in show command?